frontend/pages/policies/ManagePoliciesPage/_styles.scss

```diff
@@ -184,10 +184,33 @@
               display: flex; // required for inline icon
               gap: $pad-xsmall;
 
-              .tooltip-base {
+              // Underlines only the name text
+              &:hover {
+                text-decoration: none;
+
+                .policy-name-text {
+                  text-decoration: underline;
+                }
+              }
+
+              .critical-badge,
+              .inherited-badge {
                 display: inline-flex;
               }
 
+              .inherited-badge {
```

Comment:
let's make this a mixin so I can apply it to the queries page too?

Reply:
or better, make the badge a component? I'll do that if I have time today.

Reply:
Component in the [merge queries PR](https://github.com/fleetdm/fleet/pull/18596). Maybe merge that first and use it here?